tests/apis/platformvm/api.test.ts

```diff
@@ -2404,13 +2410,13 @@ describe("PlatformVMAPI", (): void => {
         )
       }
 
-      let staketotal: BN = new BN(0)
+      //let staketotal: BN = new BN(0)
 
-      for (let i: number = 0; i < txu1Stake.length; i++) {
-        staketotal = staketotal.add(
-          (txu1Stake[i].getOutput() as AmountOutput).getAmount()
-        )
-      }
+      //for (let i: number = 0; i < txu1Stake.length; i++) {
+      //  staketotal = staketotal.add(
+      //    (txu1Stake[i].getOutput() as AmountOutput).getAmount()
+      //  )
+      //}
```

Comment:
Same for this comment. Is it necessary to leave it in?

Reply:
Should remain to replace them with LockedIn / LockedOut logic as soon they are in.